build.gradle.kts

```diff
@@ -6,7 +6,7 @@ import org.jetbrains.kotlin.gradle.dsl.JvmTarget
 import org.jetbrains.kotlin.gradle.dsl.KotlinVersion
 
 plugins {
-    `kotlin-dsl`
+    kotlin("jvm") version "1.8.22"
```

Comment:
```suggestion
    id("org.jetbrains.kotlin.jvm") version "1.8.22"
```
(renovate)

Reply:
Good point - I just did a quick search and it should still work using this notation just FYI for the future: https://github.com/renovatebot/renovate/issues/7777

I'll change it though to be safe not sorry.